src/surfaces/drawer/drawerTitleBar.d.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export interface DrawerTitleBarPropTypes {
+    /**
+     * Add additional classes to the TitleBar.
+     */
+    className?: string;
+    closeButton?: React.ReactNode;
+    /**
+     * Supply any inline styles to the Close Button.
+     */
+    closeButtonStyle?: object;
+    /**
+     * Used for DOM testing. https://testing-library.com/docs/queries/bytestid/
+     */
+    dataTestId?: string;
+    style?: React.CSSProperties;
+    title?: React.ReactNode | string;
+    titleStyle?: object;
```

Comment:
Should probably be `React.CSSProperties`

Reply:
Fixed.

Reply:
https://github.com/saddlebackdev/react-cm-ui/pull/346/commits/373dffa65bef7979afbf129396dec1f44990425d#diff-6e938ef2f2e03f82a565b8f1c733f0d6e5fdb2fa5522fdfbda2b4ac2f61b5c9dL19-R19
Looks good.  Resolving.